.github/workflows/bashisms.yml

```
@@ -19,9 +19,8 @@ jobs:
       - uses: actions/checkout@v2
       
       - name: Install devscripts
-        run: sudo apt install devscripts
+        run: sudo apt-get update && sudo apt-get install devscripts
```

No need to update here. Containers provided by GitHub are already pre-updated.
```suggestion
        run: sudo apt install devscripts
```

---

Please check with https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners

---

No problem thank you for taking a look